src/libraries/System.Private.CoreLib/src/System/Buffers/Text/Base64Encoder.cs

```diff
@@ -85,6 +85,15 @@ public static unsafe OperationStatus EncodeToUtf8(ReadOnlySpan<byte> bytes, Span
                             goto DoneExit;
                     }
 
+                    end = srcMax - 48;
+                    if (AdvSimd.Arm64.IsSupported && (end >= src))
+                    {
+                        AdvSimdEncode(ref src, ref dest, end, maxSrcLength, destLength, srcBytes, destBytes);
+
+                        if (src == srcEnd)
+                            goto DoneExit;
+                    }
+
                     end = srcMax - 16;
                     if ((Ssse3.IsSupported || AdvSimd.Arm64.IsSupported) && BitConverter.IsLittleEndian && (end >= src))
```

Comment:
I'm not sure if we should remove the AdvSimd check here.

With this PR, it will use the vector128 version if the buffer length is `<48 && >16`.

That's probably the best option for speed, but results in a bigger library.